### PR TITLE
Don't request domain name if we already have it

### LINKED ISSUE #82: Configuration item domainQualifier: true causes wrong formatted metric
Hi,

I am using the weblogic-monitoring-exporter 1.1.1, everything works fine unless I am setting the configuration item `domainQualifier: true` in my yaml file. As soon as I define `domainQualifier: true` the weblogic-monitoring-exporter is generating a wrong metric:
`name{domain="demo_domain"} demo_domain
wls_server_activation_time{domain="demo_domain",location="DemoAdminServer"} 1581419617765
wls_server_admin_server_listen_port{domain="demo_domain",location="DemoAdminServer"} 7001
wls_server_open_sockets_current_count{domain="demo_domain",location="DemoAdminServer"} 4
wls_server_state_val{domain="demo_domain",location="DemoAdminServer"} 2`

The first line `name{domain="demo_domain"} demo_domain` is causing following error in Prometheus: `level=warn ts=2020-02-11T11:17:44.919Z caller=scrape.go:930 component="scrape manager" scrape_pool=WebLogic target=http://172.17.0.1:7001/wls-exporter/metrics msg="append failed" err="unsupported character in float"`.
The afterwards following metric lines are containing the domain as expected.

Cheers
Dirk

### PR DESCRIPTION
**Pull request by russgold**
Fixes #82 

The exporter was requesting the domain name on each view, and the logic that hid it was confused. We now only request it once, as it should not change while the server is running.

### CODE REVIEW

File: src/test/java/io/prometheus/wls/rest/WebClientImplTest.java [hunk 1]
```diff
@@ -31,7 +34,10 @@
     private static final char QUOTE = '"';
 
     /** A URL with a host guaranteed not to exist. */
-    private static final String UNDEFINED_HOST_URL = "http://mxyptlk/";
+    private static final String UNDEFINED_HOST_URL = "http://seriously-this-should-not-exist/";
```
Comment (alai8) [inline src/test/java/io/prometheus/wls/rest/WebClientImplTest.java]: Perhaps this RFC maybe helpful: https://tools.ietf.org/html/rfc2606

File: src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java [hunk 2]
```diff
@@ -1,17 +1,23 @@
 package io.prometheus.wls.rest.domain;
 /*
- * Copyright (c) 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+ * Copyright (c) 2017, 2020, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (alai8) [inline src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java]: I think we need to remove "All rights reserved" according to a review comment from Mark (https://github.com/oracle/weblogic-monitoring-exporter/pull/86#discussion_r395182828)

Comment (markxnelson) [reply src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java]: yes that is right, any file you update in 2020, you should remove the "all rights reserved"

File: src/test/java/io/prometheus/wls/rest/WebClientImplTest.java (hunk 1 above)
Comment (russgold) [reply src/test/java/io/prometheus/wls/rest/WebClientImplTest.java]: Very helpful, thanks

File: src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java (hunk 2 above)
Comment (russgold) [reply src/main/java/io/prometheus/wls/rest/domain/ExporterConfig.java]: Done.

**Pull request merged**